codalab/objects/dashboard.ws

```diff
@@ -1,18 +1,23 @@
-Welcome to your **CodaLab Dashboard**, which shows worksheets and bundles
-(programs and datasets) owned by you.  Read the
-[documentation](https://github.com/codalab/codalab-worksheets/wiki) to learn
-more.
+Welcome to your **CodaLab Dashboard**, which shows worksheets and bundles (programs and datasets) owned by you.  Read the [documentation](https://github.com/codalab/codalab-worksheets/wiki) to learn more. See [this page](https://github.com/codalab/codalab-worksheets/wiki/Worksheet-Markdown) to learn more about CodaLab’s markdown syntax. 
```

Comment:
In general, I prefer breaking this into multiple lines so that diffs are easier to see.